Enterprise-CMCS/macpro-mdct-mfp
# WP Copy Over

### Description

What's better than finishing a form? Not having to completely fill a second one from scratch. This branch contains the base for WP copy over. 

Does not cover:
- Repeating Quarter logic
- Close Out

https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/127151429/d04416e6-a7a7-4721-b485-90c9f6059197


### Related ticket(s)


CMDCT-2242
CMDCT-2304
CMDCT-2361

---
### How to test

1) Sign into MFP as a stateuser 
2) Create, fill out and submit a WP
3) Signout and then sign in as Admin
4) Approve the WP you just entered
5) Signout and signin as the stateuser again
6) Go to WP dashboard, text should update on the dashboard to indicate you can copy the previous WP
7) Copy previous WP
8) A newly created wp will be made with all the data from your previous WP

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
